test/clt-tests/core/test-equests-with-knn.rec

```diff
@@ -0,0 +1,65 @@
+––– block: ../base/start-searchd –––
+––– input –––
+mysql -h0 -P9306 -e "CREATE TABLE knn_test (id BIGINT, model TEXT, vector float_vector)"
```

Comment:
we usually check the status code in case we do not have any output

Reply:
Added status code check to KNN test commands